docs/integrations/index.md

```diff
@@ -55,6 +55,18 @@ Learn more about metadata stores [here](../learn/metadata-stores.md).
 
     [:octicons-arrow-right-24: API docs](../reference/api/metadata-stores/ibis/clickhouse.md)
 
+-   :custom-postgresql:{ .lg .middle } [**PostgreSQL**](metadata-stores/databases/postgres.md)
+
+    ---
+
+    :material-tag-outline: Database
+
+    Use PostgreSQL as a transactional metadata store with strong consistency guarantees.
```

Comment:
I think this is strange to highlight here since Metaxy is not using any transactions (except just inserting data).   
  
I'd rather word it just as "Use PostgreSQL to store Metaxy metadata."